python/lsst/analysis/tools/tasks/refCatSourcePhotometricAnalysis.py

```diff
@@ -30,11 +30,15 @@
 class RefCatSourcePhotometricAnalysisConnections(
     AnalysisBaseConnections,
     dimensions=("visit",),
-    defaultTemplates={"outputName": "sourceTable_visit_ps1_pv3_3pi_20170110_match_photom"},
+    defaultTemplates={
+        "targetCatalog": "sourceTable_visit",
+        "refCatalog": "ps1_pv3_3pi_20170110",
+        "outputName": "sourceTable_visit_ps1_pv3_3pi_20170110_match_photom",
```

Comment:
Oh is there a way that the output name (where is this used, in the parent?) can use the templates?

Reply:
I _think_ the answer is no, but I wish it were otherwise.  This is my first encounter with a lot of this stuff so I'm not certain.